EventFilter/GEMRawToDigi/test/runGEMUnpacker_cfg.py

```diff
@@ -142,9 +148,16 @@
 # Other statements
 from Configuration.AlCa.GlobalTag import GlobalTag
 process.GlobalTag = GlobalTag(process.GlobalTag, '112X_dataRun3_Prompt_v5', '')
-## for the time being the mapping does not work with the data label. Use MC instead
-if options.useB904Data:
-    process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2022_realistic', '')
+
+# Mapping for b904 GEM-CSC integration stand
+if (options.useB904GE11Long or options.useB904GE11Short):
+    process.GlobalTag.toGet = cms.VPSet(
+            cms.PSet(record = cms.string("GEMChMapRcd"),
+                     tag = cms.string("GEMChMapRcd"),
+                     connect = cms.string("sqlite_fip:EventFilter/GEMRawToDigi/test/GEMeMap_GE11_b904.db")
```

Comment:
this is under test, so maybe it was just something temporary, but we just updated the GEM ChMap in the GT, how is this tag different? @hyunyong maybe?

Reply:
This is for testing the trigger features in the b904 laboratory, where we have cosmic stands that require dedicated mapping files. This map file is indeed in line with the new GEM ChMap format

Reply:
This file is utilized for a cosmic stand in b904 laboratory, where we validate the FW features. The mapping follows the format of the recently introduced GEM ChMap in the GT. Hope this answers the question, otherwise maybe @yeckang could know more.

Reply:
@tvami
This unpacker is not for the P5. The data should come from the local setup in B904. And as you can expect, the mapping would be different from the one that we are using for the P5.

Reply:
Ok that's good to know. But then this tag should be uploaded to the DB as a single-iov tag, with an appropriate naming, and then ES prefer that. tagging @hyunyong again for doing the upload